concent_api/verifier/tasks.py

```diff
@@ -119,12 +127,12 @@ def blender_verification_order(
             unpack_archive(
                 os.path.basename(file_path)
             )
-        except (OSError, BadZipFile) as e:
+        except (OSError, BadZipFile) as exception:
```

Comment:
@cameel comment from https://github.com/golemfactory/concent/pull/514#discussion_r195779279
> What kind of OSError are you expecting here? I think you should crash on OSError.

Any problem that is related with reading file from disk. I have followed info from issue that if archive file can't be extracted (in this case, it can't too) verification should end here.

Reply:
By "archive can't be extracted" I mean a damaged archive. I think `BadZipFile` covers that. Not stuff like permission errors, missing files, etc. Those things should crash the verifier.